sdk/datafactory/azure-resourcemanager-datafactory/pom.xml

```diff
@@ -51,6 +51,12 @@
       <artifactId>azure-core-management</artifactId>
       <version>1.5.1</version> <!-- {x-version-update;com.azure:azure-core-management;dependency} -->
     </dependency>
+    <dependency>
```

Comment:
why we need junit-jupiter-engine?

Reply:
It is included in azure-core-test as test dependency, hence does not available to this lib via transient dependency.